inference-engine/src/transformations/include/transformations/common_optimizations/broadcast_elementwise_fusion.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+#include <memory>
+
+#include <transformations_visibility.hpp>
+
+#include <ngraph/ngraph.hpp>
+#include <ngraph/pass/graph_rewrite.hpp>
+#include "ngraph/pattern/matcher.hpp"
+
+namespace ngraph {
+namespace pass {
+
+class TRANSFORMATIONS_API BroadcastElementwiseFusion;
+
+}  // namespace pass
+}  // namespace ngraph
+
+/**
+ * @ingroup ie_transformation_common_api
+ * @brief remove Broadcast op before Elementwise
```

Comment:
The comment is confusing. The transformation should not always remove the Broadcast. It should do this only iff some specific conditions are met. Please, describe them in the comment.